src/Centreon/Application/Controller/UserController.php

```diff
@@ -93,7 +93,8 @@ public function getUserParameters(): View
             'locale' => $user->getLocale(),
             'is_admin' => $user->isAdmin(),
             'use_deprecated_pages' => $user->isUsingDeprecatedPages(),
-            'is_export_button_enabled' => $user->isOneClickExportEnabled()
+            'is_export_button_enabled' => $user->isOneClickExportEnabled(),
+            'default_page' => $user->getDefaultPage() !== null ? $user->getDefaultPage()->getRedirectionUri() : null
```

Comment:
Since php 8.0, you can use optional chaining : https://php.watch/versions/8.0/null-safe-operator

Reply:
done